src/common/telemetry/lib/parameterSets.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef PARAMETER_SETS_HPP
+#define PARAMETER_SETS_HPP
+
+#include <string>
+#include <set>
+#include <unordered_map>
+
+namespace Telemetry
+{
+
+// Define parameter sets for each EventWrite_ method
+const std::set<std::string> BASELINE_COMPLETE_REQUIRED_PARAMS = {
+    "DistroName",
+    "BaselineName",
+    "Mode",
+    "DurationSeconds",
+    "CorrelationId",
+    "Timestamp"    // TODO:  include common params
+};
+
+const std::set<std::string> BASELINE_COMPLETE_OPTIONAL_PARAMS = {
+    "Version"
+};
```

Comment:
Where is the ErrorEvent? The main one that will map to the 600 places? Was that going to be added later?